### PR TITLE
Pijul: Add version 1.0.0-beta

### PR DESCRIPTION
**Pull request by nightscape**
Pijul is a free and open source (GPL2) distributed version control system. Its distinctive feature is to be based on a theory of patches, while still being fast and scalable. This makes it easy to learn and use, without any compromise on power or features.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/pijul.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.0.0-beta",
+    "description": "Pijul is a free and open source (GPL2) distributed version control system. Its distinctive feature is to be based on a theory of patches, while still being fast and scalable. This makes it easy to learn and use, without any compromise on power or features.",
```
Comment (rashil2000) [inline bucket/pijul.json:3]:
```suggestion
    "description": "A fast and scalable distributed version control system, based on a theory of patches.",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
You need to use architecture 64bit block

**Comment by tech189** [comment]
Here's the 64bit architecture block added to `url`, `hash` and `autoupdate`:
```json
{
    "version": "1.0.0-beta",
    "description": "A fast and scalable distributed version control system, based on a theory of patches.",
    "homepage": "https://pijul.org",
    "license": "GPL-2.0-only",
    "architecture": {
        "64bit": {
            "url": "https://github.com/boringcactus/pijul-windows-builds/releases/download/1.0.0-beta/pijul.exe",
            "hash": "99192ae7b358c2a6c1a6b92a1da09bff5d8d0f85db8cb4aa050e83f922b6112d"
        }
    },
    "bin": "pijul.exe",
    "checkver": {
        "github": "https://github.com/boringcactus/pijul-windows-builds/",
        "re": "(?<version>[\\d\\w.-]+)/pijul\\.exe"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/boringcactus/pijul-windows-builds/releases/download/$version/pijul.exe"
            }
        }
    }
}
```

**Comment by nightscape** [comment]
Sorry, my family and me caught covid. I'll update the PR once I'm back on track.

**Comment by rashil2000** [comment]
No worries @nightscape, I made the changes.

Please take care.

**Comment by nightscape** [comment]
Thank you @rashil2000 ❤️

**Pull request merged**